ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -268,8 +267,8 @@ def test_numpy_cov(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
-        axis=axis,
         dtype=dtype[0],
+        keepdims=keep_dims,
```

Comment:
Are those changes intentional? if not we should revert them.

Reply:
While I was testing nanpercentile, the numpy_cov tests were failing because of some error related to axis (not sure if it is an unexpected parameter since the cov function doesn't take in axis)

Reply:
Well in that case, we should make a PR for fixing the other tests just to keep things clean and easy to track.